src/home_robot_sim/home_robot_sim/env/habitat_ovmm_env/habitat_ovmm_env.py

```diff
@@ -79,22 +66,41 @@ def __init__(self, habitat_env: habitat.core.env.Env, config):
                 ),
                 sem_gpu_id=(-1 if config.NO_GPU else self.habitat_env.sim.gpu_device),
             )
+        self._last_habitat_obs = None
+
+    def set_vis_dir(self):
+        scene_id = (
+            self.habitat_env.current_episode().scene_id.split("/")[-1].split(".")[0]
+        )
+        episode_id = self.habitat_env.current_episode().episode_id
+        self.visualizer.set_vis_dir(scene_id=scene_id, episode_id=episode_id)
 
     def reset(self):
         habitat_obs = self.habitat_env.reset()
+        self._last_habitat_obs = habitat_obs
         self.semantic_category_mapping.reset_instance_id_to_category_id(
             self.habitat_env
         )
         self._last_obs = self._preprocess_obs(habitat_obs)
         self.visualizer.reset()
+        self.set_vis_dir()
+        return self._last_obs
+
+    def update_hab_pose(self, hab_pose):
```

Comment:
what does this do?

Reply:
This is for supporting the change in XYZ convention when calculating tilt from pose using pytorch3d's function. Renamed to `convert_pose_to_real_world_axis`